js/script.js

```diff
@@ -76,6 +80,21 @@ const createTaskCard = (task) => {
   });
   taskCard.appendChild(deleteButton);
 
+  const checkButton = document.createElement("button");
+  checkButton.className = "btn-check";
+  checkButton.textContent = "✔️";
+  checkButton.addEventListener("click", () => {
+    task.isCompleted = !task.isCompleted;
+    taskCard.classList.toggle("isCompleted");
+    renderTasks();
+  });
+  taskCard.appendChild(checkButton);
+
+  const completedDiv = document.createElement("div");
+  completedDiv.textContent = "Completed";
+  completedDiv.style.display = task.isCompleted ? "block" : "none";
```

Comment:
it's convention not to use styling css in JS. Rather you can use class. This is because JS loads at the end of the page. So until loading JS, the style from CSS will be visible to user and then with a flick styling from JS will be rendered at the end. like you added some styling for tab using media queries in js, when you load the page initially you'll see the default style for the page, though you are in tab device. And after a while when js loaded you'll see the desired style.